### ISSUE TITLE
Add support for HES Hog series consoles

### ISSUE THREAD

**Issue by mekristo1** [benkuper/Chataigne#165]
Hi!

Great work, amazing comprehensive interface!

It would be amazing if you could build a module of basic OSC commands for the Hog4 series of lighting consoles, we use streamdeck with these boards all over.

I'd be more thank happy to beta test any module for you

https://www.etcconnect.com/webdocs/Controls/HOG/HTML/en/sect-osc_mappings.htm

**Comment by benkuper** [comment]
Hello, you can create your own custom module without needing to modify the core code.
You can also ask on the discord community server so other people can be interested and join on the project.
The github issues are focused on core code bugs and improvements.

Thank you !

**Comment by pmjcontroller1** [comment]
@mekristo1  
hello, we are interested in creating that module, but we don't have hog hardware, we can make and work together for you to do the tests.  If you are interested let me know and leave me a contact to communicate

**Comment by benkuper** [comment]
I don't have hog hardware as well. I don't understand, you want to create a module for hardware you don't have ??

**Comment by pmjcontroller1** [comment]
Yes, because I use it with my controllers in shows that provide me with hog hardware but I don't have a physical one to verify that the osc messages and communication is correct, I already have one made for chamsys and it would cost me little to adapt it

⁣
paco Lop
Dpto comercial
+34 663390200
www pmjcontroller.es​

**Comment by pmjcontroller1** [comment]
@mekristo1 Good morning.  
I have a first version of the module, please contact me to carry out the tests
